integration-test/src/main/java/com/sequenceiq/it/cloudbreak/testcase/e2e/sdx/SdxResizeTests.java

```diff
@@ -55,16 +55,16 @@ public void testSDXResize(TestContext testContext) {
                 .withClusterShape(SdxClusterShape.CUSTOM)
                 .when(sdxTestClient.createInternal(), key(sdx))
                 .await(SdxClusterStatusResponse.RUNNING, key(sdx))
-                .awaitForHealthyInstances()
```

Comment:
I think you can leave the `awaitForHealthyInstances` here, because of this is reasonable here. After we were waiting for the SDX `cluster status` (RUNNING) we should wait for the `HEALTHY instance statuses` as well. If all the instances are in healthy state and we got back CM is up and running (cluster status is RUNNING) than we can say everything is OK to say SDX is in good health to go on to the rest of the testing steps.

Reply:
reverted this change.